charts/opentelemetry-operator/templates/prometheusrule.yaml

```diff
@@ -32,7 +32,7 @@ spec:
         {{- end }}
       annotations:
         description: '{{`Reconciliation errors for {{ $labels.controller }} is increasing and has now reached {{ humanize $value }} `}}'
-        runbook_url: 'Check manager logs for reasons why this might happen'
+        runbook_url: 'https://github.com/open-telemetry/opentelemetry-operator/blob/main/docs/runbooks.md#reconcileerrors'
```

Comment:
Can we make this configurable in `values.yaml`?

Reply:
yes, done.